config/initializers/cors.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+Rails.application.config.middleware.insert_before 0, Rack::Cors do
+  allow do
+    origins '*'
```

Comment:
As we're using the `*` here, meaning that we're accepting all the calls, the updates from the other environments can be dropped, wdyt @SergiuLupaiescu?